legislation.openfisca.fr/legislation.openfisca.fr.conf

```diff
@@ -0,0 +1,27 @@
+server {
+        listen 80;
+        server_name legislation.openfisca.fr;
+
+        location /.well-known {
+            root /tmp/renew-webroot;
+        }
+
+        location / {
+                return 301 https://$server_name$request_uri;
+        }
+}
+
+server {
```

Comment:
I don't think you need to specify ssl nor logs here, as you're redirecting. Try

```
server {
    listen 443 ssl;
    server_name legislation.openfisca.fr;

    location / {
        return 301 https://fr.openfisca.org/legislation$request_uri;
    }
}

Reply:
<img width="1678" alt="capture d ecran 2018-04-09 a 11 00 55" src="https://user-images.githubusercontent.com/13916213/38489027-78d1e9a2-3be5-11e8-81ba-2ed868ba7400.png">